### PR TITLE
TinyCLR tutorials.

### PR DESCRIPTION
**Pull request by greg-norris**
Tested Code and made some typo changes. Started the TinyCLR getting started section.

### CODE REVIEW

File: tinyclr/tutorials/intro.md [hunk 1]
```diff
@@ -1,3 +1,45 @@
...
+
+Extract all the files from the downloaded zip into a folder on your desktop. 
+
+Next run the VSIX installer file, found inside the folder you just extracted to you DESKTOP
+
+
+
+## Adding Nuget Packages
+
+For the current preview releases you'll have to create a local NUGET folder to host the NUGET packages that are found in the [TinyCLR OS Download](http://ghielectronics.com/downloads/TinyCLR/TinyCLR.0.4.0.zip)
```
Comment (Arke64) [inline tinyclr/tutorials/intro.md]: NUGET should be NuGet. There are a few occurrences.

File: tinyclr/tutorials/intro.md [hunk 2]
```diff
@@ -1,3 +1,45 @@
 # TinyCLR Tutorials
 
 Here you will find tutorials covering different features of the TinyCLR OS. We try to keep the documents hardware-generic as much as possible but when we can't, we use [FEZ](../../hardware/fez.md) as a reference.
+___
```
Comment (Arke64) [inline tinyclr/tutorials/intro.md]: What is this for?

File: tinyclr/tutorials/intro.md [hunk 3]
```diff
@@ -1,3 +1,45 @@
...
+
+* [Visual Studio 2017 Download](https://www.visualstudio.com/downloads/)
+
+Next you have to download the lastest version of TinyCLR OS here:
+
+ 
+* [TinyCLR OS Download](http://ghielectronics.com/downloads/TinyCLR/TinyCLR.0.4.0.zip)
+
+Release notes:
+* [TinyCLR OS Release Notes](https://www.ghielectronics.com/docs/350/tinyclr-os)
```
Comment (Arke64) [inline tinyclr/tutorials/intro.md]: This should point to release notes under docs.ghi....

File: tinyclr/tutorials/intro.md [hunk 4]
```diff
@@ -1,3 +1,45 @@
...
+
+## Getting Started
+First you'll need to download and install Visual Studio 2017, if you already haven't 
+
+* [Visual Studio 2017 Download](https://www.visualstudio.com/downloads/)
+
+Next you have to download the lastest version of TinyCLR OS here:
+
+ 
+* [TinyCLR OS Download](http://ghielectronics.com/downloads/TinyCLR/TinyCLR.0.4.0.zip)
```
Comment (Arke64) [inline tinyclr/tutorials/intro.md]: Make the link invalid so we know to go back and update it

File: tinyclr/tutorials/intro.md [hunk 5]
```diff
@@ -1,3 +1,45 @@
...
+Next run the VSIX installer file, found inside the folder you just extracted to you DESKTOP
+
+
+
+## Adding Nuget Packages
+
+For the current preview releases you'll have to create a local NUGET folder to host the NUGET packages that are found in the [TinyCLR OS Download](http://ghielectronics.com/downloads/TinyCLR/TinyCLR.0.4.0.zip)
+
+In the near future all libraries will be downloaded through NuGet. Because of this, there will be no SDK to install or machines to lock to specific versions. Simply use NuGet to download any needed version of any needed library on a per-project basis.
+___
```
Comment (Arke64) [inline tinyclr/tutorials/intro.md]: Same as above

File: tinyclr/tutorials/intro.md [hunk 6]
```diff
@@ -1,3 +1,45 @@
...
+
+## Create a New Project
+Open Visual Studio, select FILE>New Project. There should be a "TinyCLR" option under "Visual C#"
```
Comment (Arke64) [inline tinyclr/tutorials/intro.md]: Make it "select `File > New Project`. There should be a `TinyCLR` option under `Visual C#`.

This is what the backtick is for.

File: tinyclr/tutorials/intro.md [hunk 7]
```diff
@@ -1,3 +1,45 @@
...
+
+Click the "OK" button a project and program are created. The project has only one C# file, called Program.cs. C# source files are listed in the "Solution Explorer" window. If the Solution Explorer is not opened, use the VIEW->Solution Explorer menu.
+
```
Comment (Arke64) [inline tinyclr/tutorials/intro.md]: Remove all these newlines.

File: tinyclr/tutorials/intro.md [hunk 8]
```diff
@@ -1,3 +1,45 @@
...
+For the current preview releases you'll have to create a local NUGET folder to host the NUGET packages that are found in the [TinyCLR OS Download](http://ghielectronics.com/downloads/TinyCLR/TinyCLR.0.4.0.zip)
+
+In the near future all libraries will be downloaded through NuGet. Because of this, there will be no SDK to install or machines to lock to specific versions. Simply use NuGet to download any needed version of any needed library on a per-project basis.
+___
+
+## Create a New Project
+Open Visual Studio, select FILE>New Project. There should be a "TinyCLR" option under "Visual C#"
+
+Click the "OK" button a project and program are created. The project has only one C# file, called Program.cs. C# source files are listed in the "Solution Explorer" window. If the Solution Explorer is not opened, use the VIEW->Solution Explorer menu.
```
Comment (Arke64) [inline tinyclr/tutorials/intro.md]: Same as above, use backtick.

File: tinyclr/tutorials/intro.md (hunk 2 above)
Comment (greg-norris) [inline tinyclr/tutorials/intro.md]: It creates a small grey bar that spans the document like a divider.

**Pull request closed without merging**